libpod/container_internal_linux.go

```diff
@@ -184,11 +184,11 @@ func (c *Container) prepare() error {
 func (c *Container) isWorkDirSymlink(resolvedPath string) bool {
 	// We cannot create workdir since explicit --workdir is
 	// set in config but workdir could also be a symlink.
-	// If its a symlink lets check if resolved link is present
+	// If it is a symlink let us check if the resolved link is present
```

Comment:
I'm having trouble understanding all this; it might need more than just minor edits. Maybe:
```
// If it's a symlink, check if the resolved target is present in the container.
// If so, that's a valid use case: return nil.
```
(note that `use-case` below is improperly hyphenated)

Reply:
I changed to your version